geode-redis/README.md

```diff
@@ -128,210 +128,98 @@ not connected>
-|                    	|                                                    	| ZPOPMAX                       	|
-|                    	|                                                    	| ZPOPMIN                       	|
-|                    	|                                                    	| ZRANGE                        	|
-|                    	|                                                    	| ZRANGEBYLEX                   	|
-|                    	|                                                    	| ZRANGEBYSCORE                 	|
-|                    	|                                                    	| ZRANK                         	|
-|                    	|                                                    	| ZREM                          	|
-|                    	|                                                    	| ZREMRANGEBYLEX                	|
-|                    	|                                                    	| ZREMRANGEBYRANK               	|
-|                    	|                                                    	| ZREMRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANGE                     	|
-|                    	|                                                    	| ZREVRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANK                      	|
-|                    	|                                                    	| ZSCAN                         	|
-|                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
-
-**NOTES:**
-
-[1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
-[2] - SLOWLOG is implemented as a NoOp.
+The Geode APIs compatible with Redis implement a subset of the full Redis command set.
+
+#### <a name="supported-redis-commands"></a> Supported Redis Commands [Return to top](#introduction)
+- APPEND 
+- AUTH
+- DECR
+- DECRBY
+- DEL
+- EXISTS
+- EXPIRE
+- EXPIREAT
+- GET
+- GETRANGE
+- HDEL
+- HEXISTS
+- HGET
+- HGETALL
+- HINCRBY
+- HINCRBYFLOAT
+- HLEN
+- HMGET
+- HMSET
+- HSCAN
+- HSET
+- HSETNX
+- HSTRLEN
+- HVALS
+- HKEYS
+- INCR
+- INCRBY
+- INCRBYFLOAT
+- INFO <sup>1</sup>
+- KEYS
+- MGET
+- PERSIST
+- PEXPIRE
+- PEXPIREAT
+- PING
+- PSUBSCRIBE
+- PTTL
+- PUBLISH
+- PUNSUBSCRIBE
+- QUIT
+- RENAME
+- SADD
+- SET
+- SETNX
+- SLOWLOG <sup>2</sup>
+- SMEMBERS
+- SREM
+- STRLEN
+- SUBSCRIBE
+- TTL
+- TYPE
+- UNSUBSCRIBE
+
+**NOTES:** <br/>
+<sup>1</sup> INFO is implemented for the sections and fields listed below:
+ - clients
+    - connected_clients
+    - blocked_clients (will always return 0)
+ - cluster
+    - cluster_enables (will always return 0)
+ - keyspace
+    - returns stats for db: 0
```

Comment:
Not sure if we want to mention that it will return `keys`, but `expires` and `avg_ttl` will always be 0? Also, it seems to be referenced in Redis docs and the keyspace section as `db0`.

Reply:
thx